internal/services/oracle/oracle_database_system_resource.go

```diff
@@ -0,0 +1,496 @@
+	DiskRedundancy               string                       `tfschema:"disk_redundancy"`
+	DisplayName                  string                       `tfschema:"display_name"`
+	Domain                       string                       `tfschema:"domain"`
+	InitialDataStorageSizeInGb   int64                        `tfschema:"initial_data_storage_size_in_gb"`
+	NodeCount                    int64                        `tfschema:"node_count"`
+	PluggableDatabaseName        string                       `tfschema:"pluggable_database_name`
+	StorageVolumePerformanceMode string                       `tfschema:"storage_volume_performance_mode"`
+	TimeZone                     string                       `tfschema:"time_zone"`
+}
+
+func (DatabaseSystemResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.DatabaseSystemName,
+		},
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		"location": commonschema.Location(),
+
+		// Required
+		"admin_password": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			Sensitive:    true,
+			ForceNew:     true,
+			ValidateFunc: validate.DatabaseSystemPassword,
+		},
+
+		"compute_count": {
+			Type:         pluginsdk.TypeInt,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.FloatBetween(1.0, 512.0),
+		},
+
+		"compute_model": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringInSlice(dbsystems.PossibleValuesForComputeModel(), false),
+		},
+
+		"database_edition": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringInSlice(dbsystems.PossibleValuesForDbSystemDatabaseEditionType(), false),
+		},
+
+		"database_version": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+
+		"hostname": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.DatabaseSystemName,
+		},
+
+		"license_model": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringInSlice(dbsystems.PossibleValuesForLicenseModel(), false),
+		},
+
+		"network_anchor_id": commonschema.ResourceIDReferenceRequiredForceNew(&networkanchors.NetworkAnchorId{}),
+
+		"resource_anchor_id": commonschema.ResourceIDReferenceRequiredForceNew(&resourceanchors.ResourceAnchorId{}),
+
+		"shape": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringInSlice(dbversions.PossibleValuesForBaseDbSystemShapes(), false),
+		},
+
+		"source": {
```

Comment:
`None` value should not be exposed. ([ref](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/schema-design-considerations.md#the-none-value-or-similar))